kylin-spark-project/kylin-spark-common/src/main/java/org/apache/kylin/engine/spark/metadata/cube/PathManager.java

```diff
@@ -58,12 +58,11 @@ public static String getSegmentParquetStoragePath(String hdfsWorkDir, String cub
     /**
      * Delete segment path
      */
-    public static boolean deleteSegmentParquetStoragePath(CubeInstance cube, CubeSegment segment) throws IOException {
-        if (cube == null || segment == null) {
+    public static boolean deleteSegmentParquetStoragePath(CubeInstance cube, String segmentName, String identifier) throws IOException {
+        if (cube == null) {
```

Comment:
it'd better check whether these two parameters 'segmentName' and 'identifier' are valid